xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream/stream_codec.h

```diff
@@ -141,6 +141,28 @@ extern "C"
     /// [chroma subsampling](https://en.wikipedia.org/wiki/Chroma_subsampling): 4:2:0, 4:2:2, 4:4:4
     /// see [AV1 specification](https://aomedia.org/av1/specification/)
     AV1CodecProfileProfessional,
+
+    // Audio codec profiles
+
+    AACCodecProfileMAIN,
```

Comment:
Only a bit about API descriptions missing on new added values :smirk:

Reply:
description added for each one